modules/300-prometheus/monitoring/prometheus-rules/prometheus-free-space.yaml

```diff
@@ -0,0 +1,19 @@
+- name: d8.prometheus.disk_usage
+  rules:
+  - alert: PrometheusDiskUsage
```

Comment:
We also need alerts that LongtermPrometheus has data for less than 30 days and Prometheus has data for less than 10 days.

Reply:
Added these alerts.